packages/typography/src/Label/Label.tsx

```diff
@@ -13,32 +18,36 @@ import {
 } from './Label.styles';
 import { LabelProps } from './Label.types';
 
-export const Label = ({
-  baseFontSize: baseFontSizeOverride,
-  darkMode: darkModeProp,
-  className,
-  children,
-  disabled = false,
-  ...rest
-}: LabelProps) => {
-  const baseFontSize = useUpdatedBaseFontSize(baseFontSizeOverride);
-  const { theme } = useDarkMode(darkModeProp);
+export const Label = Polymorphic<LabelProps>(
```

Comment:
Should `Description` also extend Polymorphic?

Reply:
see above